cmd/fs-v1-metadata.go

```diff
@@ -375,6 +383,8 @@ func checkFormatSanityFSV2(fsPath string) error {
 // Check for sanity of FS format for a given version.
 func checkFormatSanityFS(fsPath string, fsFormatVersion string) (err error) {
 	switch fsFormatVersion {
+	case fsFormatV1:
```

Comment:
That why it was kept there.. while it is not going to be used or doing anything in the first place.